ensembl/conf/SiteDefs.pm

```diff
@@ -32,6 +32,7 @@ sub update_conf {
   $SiteDefs::ARCHIVE_BASE_DOMAIN        = 'archive.ensembl.org';
   $SiteDefs::ENSEMBL_REST_URL           = 'https://rest.ensembl.org';  # URL for the REST API
   $SiteDefs::ENSEMBL_REST_DOC_URL       = 'https://github.com/Ensembl/ensembl-rest/wiki';
+  $SiteDefs::ENSEMBL_EXAMPLE_FTP_URL    = 'ftp://ftp.ensembl.org/pub/current_mysql';
```

Comment:
If I understand the issue, you need a parameter for the mysql FTP URL. we already have a parameter for FTP URL (ENSEMBL_FTP_URL). so maybe the new parameter should be:
ENSEMBL_MYSQL_FTP_URL = ENSEMBL_FTP_URL."/current_mysql"; 
and change it accordingly in the 37 plugins

From what i understand you should probably have two parameters: